cli/nep11_test.go

```diff
@@ -231,8 +232,13 @@ func TestNEP11_OwnerOf_BalanceOf_Transfer(t *testing.T) {
 	// tokensOf: good, several tokens
 	tokenID1 := mint(t)
 	e.Run(t, cmdTokensOf...)
-	e.checkNextLine(t, string(tokenID))
-	e.checkNextLine(t, string(tokenID1))
+	fst, snd := tokenID, tokenID1
```

Comment:
Nice fix, but I think it should be in a separate commit.

Reply:
Moved contract update before syscall removal.

Reply:
But is it really related to NFT contract changes? It looks more like a fix of its own to me.

Reply:
Earlier tokens were stored in a list (chronological order), now they are stored by key (iterated in alphabetical).